build_tools/travis/before_install.sh

```diff
@@ -87,7 +87,7 @@ else
     fi
 
     if [[ "$BUILD_DOCUMENTATION" == "ON" ]]; then
-        sudo apt-get install doxygen graphviz
+        sudo apt-get install -y doxygen graphviz gnuplot
```

Comment:
ok to merge but can you just remove gnuplot here? I don't see why this is needed. thx